networkx/algorithms/bipartite/covering.py

```diff
@@ -0,0 +1,67 @@
+"""
+********
+Covering
+********
+"""
+
+#    Copyright (C) 2015 by
+#    Nishant Nikhil <nishantiam@gmail.com>
+#    All rights reserved.
+#    BSD license.
+
+import networkx as nx
+from networkx.utils import not_implemented_for, arbitrary_element
+
+
+@not_implemented_for('directed', 'multigraph')
+def min_edge_cover(G):
+    """Returns a set of edges which constitute the minimum edge cover of the graph.
+    An edge cover of a graph is a set of edges such that every vertex of
+    the graph is incident to at least one edge of the set.
+    A minimum edge cover is an edge covering of smallest possible size.
+
+    A smallest edge cover can be found in polynomial time by finding
```

Comment:
This is information about the implementation, not the behavior. This should go in the `Notes` section; see the [numpydoc][1] recommendations for more information.

[1]: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections